db.json

```diff
@@ -0,0 +1,114 @@
+{
+  "users": [
+    {
+      "id": 1,
+      "name": "admin",
+      "password": "admin",
+      "lastName": "",
+      "image": "",
+      "allergies": "",
+      "phone": "",
+      "email": "",
+      "address": ""
+    },
+    {
+      "id": 2,
+      "name": "guest",
+      "password": "guest",
+      "lastName": "",
+      "image": "",
+      "allergies": "",
+      "phone": "",
+      "email": "",
+      "address": ""
+    }
+  ],
+  "therapies": [
+    {
+      "id": 1,
+      "name": "Therapy 1",
+      "userId": 1,
+      "contact": 2,
+      "notes": "this is the notes"
+    },
+    {
+      "id": 2,
+      "name": "Therapy 2",
+      "userId": 1,
+      "contact": 2,
+      "notes": "this is the notes"
+    }
+  ],
+  "contacts": [
+    {
+      "id": 1,
+      "name": "Sarah Davis",
+      "qualification": "Dr",
+      "phone": "+237 656585542",
+      "email": "davis@gmail.com",
+      "address": "Dschang, Cameroun",
+      "notes": ""
+    },
+    {
+      "id": 2,
+      "name": "Laura Thompson",
+      "qualification": "Dr",
+      "phone": "+237 674772917",
+      "email": "laura@gmail.com",
+      "address": "Douala, Cameroun",
+      "notes": ""
+    },
+    {
+      "id": 3,
+      "name": "John Smith",
+      "qualification": "Dr",
+      "phone": "+237 677345741",
+      "email": "john@gmail.com",
+      "address": "Yaounde, Cameroun",
+      "notes": ""
+    }
+  ],
+  "prescriptions": [
+    {
+      "id": 1,
+      "therapy": 1,
+      "medicine": 2,
+      "dateFrom": "2024-09-19",
+      "dateTo": "2024-10-30"
+    }
+  ],
+  "prescriptiontimes": [
+    {
+      "id": 5,
+      "time": "12h30PM"
```

Comment:
In my opinion, you should add a "prescription_id" property

Reply:
just in prescription ?

Reply:
this is good, ensure to design this object in a way that allows the server to be compliant with the api we designed.